moveit_ros/moveit_servo/include/moveit_servo/pose_tracking.h

```diff
@@ -187,7 +187,7 @@ class PoseTracking
   // Flag that a different thread has requested a stop.
   std::atomic<bool> stop_requested_;
 
-  double angular_error_;
+  std::shared_ptr<double> angular_error_;
```

Comment:
Any reason this is a shared pointer instead of a unique one?

Not solid on the pros/cons of each but it seems like this variable will never be shared right?

Reply:
Good point. Sounds like unique_ptr is a little more efficient, too

https://stackoverflow.com/a/22296124/3499467